packages/@uppy/companion/src/companion.js

```diff
@@ -103,7 +103,7 @@ module.exports.app = (optionsArg = {}) => {
   // override provider credentials at request time
   // Making `POST` request to the `/connect/:provider/:override?` route requires a form body parser middleware:
   // See https://github.com/simov/grant#dynamic-http
-  app.use('/connect/:oauthProvider/:override?', express.urlencoded({ extended: false }), getCredentialsOverrideMiddleware(providers, options))
+  app.use('/connect/:oauthProvider/:override', express.urlencoded({ extended: false }), getCredentialsOverrideMiddleware(providers, options))
```

Comment:
looking at https://expressjs.com/en/guide/migrating-5.html#path-syntax i think wee need this instead?

```suggestion
  app.use('/connect/:oauthProvider{/:override}', express.urlencoded({ extended: false }), getCredentialsOverrideMiddleware(providers, options))
```

source: https://www.npmjs.com/package/path-to-regexp#optional